app/main.py

```diff
@@ -1 +1,45 @@
-# write your code here
+class InvalidCommandFormat(Exception):
```

Comment:
Appreciate the effort but you don't need most of these. 
Also  it is a good practice to keep exceptions in separate module and import from there